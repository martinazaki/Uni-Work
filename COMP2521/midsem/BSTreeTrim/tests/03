9 4 2 1 3 6 5 7 8 11 10 12
3
7
