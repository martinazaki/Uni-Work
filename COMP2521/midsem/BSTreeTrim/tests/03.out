Enter the preorder traversal of the BST: Enter value of lower: Enter value of upper: BST before trimming values between 3 and 7:

           9
          / \
         /   \
        /     \
       /       \
      4        11
     / \       / \
    /   \     /   \
   /     \   10   12
  2       6
 / \     / \
1   3   5   7
             \
              8

BST after trimming values between 3 and 7:
=== ignored by test >>>

       9
      / \
     /   \
    8    11
   /     / \
  2     /   \
 /     10   12
1

<<< ignored by test ===

Inorder traversal of the BST: 1 2 8 9 10 11 12 
